tests/convolution_test.py

```diff
@@ -18,14 +18,10 @@ def test_forward(self):
         filter_shape = (h_filter, w_filter, channels_in, channels_out)
         filter_values = np.random.normal(size=filter_shape)
 
-        config = tfe.LocalConfig([
-            'server0',
-            'server1',
-            'crypto_producer'
-        ])
+        config = tfe.LocalConfig(3)
 
         # convolution pond
-        with tfe.protocol.Pond(*config.get_players('server0, server1, crypto_producer')) as prot:
+        with tfe.protocol.Pond(*config.players) as prot:
```

Comment:
Setting up the pond this way doesn't work anymore. Needs to be reverted to how it was.